packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,176 @@
+# Carousel 
+
+Carousel is a UI component that allows the display and navigation of a set of content items, typically images or cards, within a limited space. It is often used to showcase multiple pieces of content in a visually appealing and interactive way.
+
+<img src="./carousel-thumbnail.png" width="380" />
+
+## Design
+
+[Figma Link](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=2115-975416&mode=design&t=gjEux4FLvrLihxbH-0) to all variants of the Carousel component
+
+## API
+
+### Carousel
+
+| Prop                     | Type                                   | Default                                  | Description                                                                                                                                                                                                  | Required |
+| ------------------------ | -------------------------------------- | ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| children                 | `CarouselItem`                         | `undefined`                              | Carousel slides                                                                                                                                                                                              | ✅        |
+| autoPlay                 | `boolean`                              | `false`                                  | If true, the carousel will autoplay                                                                                                                                                                          |          |
```

Comment:
Should we provide delay prop to decide the delay ? Also another prop to identify if it needs to be a loop ?

Reply:
If the usecase comes in we will add a delay prop but right now we've standardized it to 6s. 

> Also another prop to identify if it needs to be a loop
The looping behaviour is standardised from design side only so probably no.